src/mrnet/stochastic/serialize.py

```diff
@@ -129,43 +130,49 @@ def __extract_index_mappings(self, reactions):
             forward_free_energy = reaction.free_energy_A
             backward_free_energy = reaction.free_energy_B
 
+            forward_rate = reaction.k_A
+            backward_rate = reaction.k_B
+
             index_to_reaction.append(
                 {
                     "reactants": reactant_indices,
                     "products": product_indices,
                     "free_energy": forward_free_energy,
+                    "rate_constant": forward_rate,
                 }
             )
             index_to_reaction.append(
                 {
                     "reactants": product_indices,
                     "products": reactant_indices,
                     "free_energy": backward_free_energy,
+                    "rate_constant": backward_rate,
                 }
             )
 
-        for reaction in index_to_reaction:
+        if use_thermo_cost:
```

Comment:
Pretty soon we're probably going to want to be able to use true barriers for some reactions and "thermo_cost" barriers for others, so this is going to need to change, but it's probably fine for now

Reply:
I don't think there's a clean way to selectively use thermo/constant barrier costs within the serialization script. Do we want the user to have to provide an array of Booleans saying which reactions should and shouldn't have thermo costs?

I think the solution here is just to create a new RateCalculator class that provides a constant barrier, and apply that to Reactions without known kinetics.